tox.ini

```diff
@@ -48,6 +48,14 @@ commands =
     pip list
     py.test --capture=no --strict {posargs}
 
+# This target downgrades cffi to the minimum supported in setup.py
```

Comment:
Please add a comment to `setup.py` saying this needs to be updated if the minimimum cffi version is changed.